auth0/src/test/java/com/auth0/android/authentication/AuthenticationAPIClientTest.java

```diff
@@ -1283,4 +1339,9 @@ public void shouldParseUnauthorizedPKCEError() throws Exception {
         }.getType();
         return gson.fromJson(request.getBody().readUtf8(), mapType);
     }
+
+    private String getDefaultLocale() {
```

Comment:
If you mean to test `getDefaultLocale()` then that's not possible with Mockito, as the `Locale` class has the _static_ & _final_ modifiers and can't be mocked.